src/web-components/components/Logo/Logo.astro

```diff
@@ -0,0 +1,64 @@
+            <path d="M110.6,121.8c-1.4-0.3-4.1-1-5.6-1.2c-26.7-3.9-40.4-7.4-59.7-0.2c18.9-31.2,9.2-73.9,2.5-94.2
+              c-0.7-2.4-0.8-2.5-10.1-22.3c-3-6.3-2.6-1.6-2.6-1.6c10.3,65.9-27.7,89-6.2,134.2c0,0-0.1,0.1-0.1,0.1c0,0,0.1,0,0.1,0
+              c0.2,0.4,0.3,0.7,0.5,1.1C15.7,149.1,6.3,162.2,0,174.2h4c9.8-18.2,22.2-30,30.7-36.6c16,2.2,42.8,2.6,70.1-6.7
+              c1.1-0.4,2.1-0.7,3.2-0.9c5.1-1.1,13.6-1.9,18.6-1.8C133.7,128.5,117.4,123.3,110.6,121.8z"/>
+            <path d="M80.9,163c0.6,0.6,0.9,1.3,0.9,2.2v38.3c0,0.9-0.4,1.6-1,2.2c-0.6,0.6-1.4,0.9-2.3,0.9c-0.8,0-1.6-0.3-2.2-0.9
+              c-0.6-0.6-0.9-1.3-0.9-2.2V187H52.9v16.4c0,0.9-0.3,1.6-1,2.2c-0.6,0.6-1.4,0.9-2.2,0.9c-0.9,0-1.7-0.3-2.3-0.9
+              c-0.6-0.6-0.9-1.3-0.9-2.2v-38.3c0-0.8,0.3-1.6,0.9-2.2c0.6-0.6,1.4-0.9,2.3-0.9c0.8,0,1.6,0.3,2.2,0.9c0.6,0.6,0.9,1.3,0.9,2.2v16
+              h22.5v-16c0-0.8,0.3-1.6,1-2.2c0.6-0.6,1.4-0.9,2.2-0.9C79.5,162.1,80.3,162.4,80.9,163z"/>
+            <path d="M186.1,195.9c-1.8,3.5-4.4,6.2-7.6,8.2c-3.3,2-6.9,3-11,3c-4.1,0-7.8-1-11-3c-3.3-2-5.8-4.7-7.6-8.2
+              c-1.8-3.5-2.8-7.3-2.8-11.6s0.9-8.1,2.8-11.6c1.8-3.5,4.4-6.2,7.6-8.2c3.3-2,6.9-3,11-3c4.1,0,7.8,1,11,3c3.3,2,5.8,4.7,7.6,8.2
+              c1.8,3.5,2.8,7.3,2.8,11.6S187.9,192.5,186.1,195.9z M180.4,175.7c-1.3-2.6-3-4.6-5.3-6c-2.3-1.5-4.8-2.2-7.7-2.2
+              c-2.9,0-5.4,0.7-7.7,2.2c-2.3,1.4-4,3.4-5.3,6c-1.3,2.6-1.9,5.4-1.9,8.7c0,3.2,0.6,6,1.9,8.6c1.3,2.6,3,4.6,5.3,6
+              c2.3,1.5,4.8,2.2,7.7,2.2c2.9,0,5.4-0.7,7.7-2.2c2.3-1.5,4-3.5,5.3-6c1.3-2.6,1.9-5.4,1.9-8.6S181.6,178.3,180.4,175.7z"/>
+            <path d="M293.7,163c0.6,0.6,0.9,1.3,0.9,2.1v38.4c0,0.8-0.3,1.6-0.9,2.2c-0.6,0.6-1.3,0.9-2.3,0.9
+              c-0.9,0-1.7-0.3-2.3-0.9c-0.6-0.6-0.9-1.3-0.9-2.2v-29.4L276,192.8c-0.3,0.4-0.7,0.7-1.1,1c-0.4,0.3-0.9,0.4-1.4,0.3
+              c-0.4,0-0.9-0.1-1.3-0.3c-0.4-0.3-0.8-0.6-1.1-1l-12.4-19.2v30c0,0.9-0.3,1.6-0.8,2.2c-0.5,0.6-1.2,0.9-2.1,0.9
+              c-0.8,0-1.5-0.3-2.1-0.9c-0.5-0.6-0.8-1.3-0.8-2.2v-38.4c0-0.8,0.3-1.5,0.9-2.1c0.6-0.6,1.3-0.9,2.2-0.9c0.5,0,1,0.1,1.5,0.4
+              c0.5,0.3,0.9,0.6,1.2,1l15,22.8l15.2-22.8c0.7-0.9,1.5-1.4,2.5-1.4C292.4,162.1,293.1,162.4,293.7,163z"/>
+            <path d="M386.1,201.6c0.6,0.6,0.9,1.3,0.9,2.1c0,0.8-0.3,1.5-0.9,2.1c-0.6,0.5-1.3,0.8-2.2,0.8h-22.1
+              c-0.8,0-1.6-0.3-2.2-0.9c-0.6-0.6-0.9-1.3-0.9-2.2v-38.3c0-0.8,0.3-1.6,0.9-2.2c0.6-0.6,1.3-0.9,2.2-0.9h22.1c0.9,0,1.6,0.3,2.2,0.8
+              c0.6,0.6,0.9,1.3,0.9,2.2c0,0.8-0.3,1.5-0.9,2.1c-0.6,0.5-1.3,0.8-2.2,0.8h-18.8v12.9h16c0.9,0,1.6,0.3,2.2,0.8
+              c0.6,0.5,0.9,1.2,0.9,2.1c0,0.9-0.3,1.6-0.9,2.1c-0.6,0.5-1.3,0.8-2.2,0.8h-16v14h18.8C384.8,200.7,385.5,201,386.1,201.6z"/>
+          </svg>
+        </div>
+        `
+      this.shadowRoot.textContent = ""
+      this.shadowRoot.appendChild(template.content.cloneNode(true))
+    }
+  }
+  customElements.define("wc-logo", Logo)
```

Comment:
The class _defines_ the functionality, and then this line defines the name to use the component.